tests/test_modules/test_model_performance/test_confusion_matrix.py

```diff
@@ -76,9 +87,22 @@ def test_confusion_matrix_reorder_rejection_class(guse_text_config):
     dm = mod.get_dataset_split_manager()
 
     # Assert that the rejection class in not the last one already
```

Comment:
```suggestion
    # Assert that the rejection class is not the last one already
```
Sorry, out of scope, but really small typo since you're working in the file...